media/js/firefox/new/scene1-macos-attribution.js

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
```

Comment:
I know there's not much code here, but seeing as it does some pretty important functionality it would be great if it had some unit tests?